packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -197,31 +256,53 @@ export const DatepickerCalendar: ComponentWithAs<'div', DatepickerCalendarProps>
             ...unhandledProps,
           })}
         >
+          {renderHeader()}
           <Grid rows={grid.length + 1} columns={DAYS_IN_WEEK}>
-            {_.times(DAYS_IN_WEEK, dayNumber => (
-              <Text
-                key={`header ${dayNumber}`}
-                align="center"
-                content={localizedStrings.shortDays[(dayNumber + firstDayOfWeek) % DAYS_IN_WEEK]}
-              />
-            ))}
-            {_.map(grid, week =>
-              _.map(week, (day: IDay) => {
-                return (
-                  <Button
-                    key={day.key}
-                    content={day.date}
-                    aria-label={`${formatMonthDayYear(day.originalDate, localizedStrings)}`}
-                    onClick={e => {
-                      _.invoke(props, 'onDateChange', e, { ...props, value: day });
-                    }}
-                    primary={day.isSelected}
-                    disabled={!day.isInMonth}
-                    text
-                  />
-                );
+            {_.times(DAYS_IN_WEEK, dayNumber =>
```

Comment:
Just out of curiosity, why use lodash instead `DAYS_IN_WEEK.map`?

Reply:
Yeah, we could use native code here instead, no specific reasons to have it done using lodash

Reply:
Asking this because I always get confused which should be used. @layershifter What should we prefer? I remember @miroslavstastny commented about reducing lodash usage.

Reply:
In this case `DAYS_IN_WEEK.map` makes sense

Reply:
Agreed offline with @layershifter to keep it like it is, since `DAYS_IN_WEEK` is a constant 7, so we would need to create a new array to call `.map`